pyppeteer/frame_manager.py

```diff
@@ -4,86 +4,208 @@
 """Frame Manager module."""
 
 import asyncio
-from collections import OrderedDict
+import re
 import logging
+from collections import OrderedDict
 from types import SimpleNamespace
-from typing import Any, Awaitable, Dict, Generator, List, Optional, Set, Union
+from typing import Any, Awaitable, Dict, List, Optional, Set, Union
 
 from pyee import EventEmitter
 
 from pyppeteer import helper
+from pyppeteer.domworld import DOMWorld
+from pyppeteer.events import Events
+from pyppeteer.jshandle import JSHandle
 from pyppeteer.connection import CDPSession
 from pyppeteer.element_handle import ElementHandle
-from pyppeteer.errors import NetworkError
-from pyppeteer.execution_context import ExecutionContext, JSHandle
-from pyppeteer.errors import ElementHandleError, PageError, TimeoutError
+from pyppeteer.errors import BrowserError
+from pyppeteer.execution_context import ExecutionContext
+from pyppeteer.errors import ElementHandleError, PageError
+from pyppeteer.lifecycle_watcher import LifecycleWatcher
+from pyppeteer.network_manager import NetworkManager
+from pyppeteer.timeout_settings import TimeoutSettings
 from pyppeteer.util import merge_dict
 
 logger = logging.getLogger(__name__)
 
+UTILITY_WORLD_NAME = '__puppeteer_utility_world__'
+EVALUATION_SCRIPT_URL = '__puppeteer_evaluation_script__'
+SOURCE_URL_REGEX = re.compile(
+    r'^[\040\t]*//[@#] sourceURL=\s*(\S*?)\s*$',
```

Comment:
IMO a space literal is a lot more readable here, I wasn't able to recall off the top of my head that \040 is a space
```suggestion
    r'^[ \t]*//[@#] sourceURL=\s*(\S*?)\s*$',
```